### PR TITLE
updated the footer to stack in mobile mode, remove the absolute posit…

### PR DESCRIPTION
**Pull request by Leprof20**
updated the footer to stack in mobile mode, removed the absolute positioning of the footer, and set the home css to 100vh (to ensure footer stays at the bottom)

 image

### CODE REVIEW

**Review by Sajakhtar — APPROVED** [review]

File: app/assets/stylesheets/pages/_home.scss [hunk 1]
```diff
@@ -1 +1,4 @@
 // Specific CSS for your home-page
+.container {
```
Comment (andrerferrer) [inline app/assets/stylesheets/pages/_home.scss:2]: careful!
this is a bootstrap class. 
it's better to rename the class to something else or use bootstrap all the way
@Leprof20 @Sajakhtar

File: app/assets/stylesheets/components/_footer.scss [hunk 2]
```diff
@@ -26,9 +25,19 @@
...
 }
 
 .footer-copyright img {
   width: 20px;
 }
+
+@media screen and ( max-width:749px ){
+
+.footer {
+    height: auto;padding: 0;
```
Comment (andrerferrer) [inline app/assets/stylesheets/components/_footer.scss:38]: careful with two statements in the same line @Leprof20

**Review by andrerferrer — COMMENTED** [review]
check comments pls @Leprof20 @Sajakhtar

File: app/assets/stylesheets/components/_footer.scss [hunk 3]
```diff
@@ -26,9 +25,19 @@
...
+  color: $light-gray;
 }
 
 .footer-copyright img {
   width: 20px;
 }
+
+@media screen and ( max-width:749px ){
+
+.footer {
```
[the remark this answers is not in the document]
Comment (Leprof20) [reply app/assets/stylesheets/components/_footer.scss:37]: Thanks, done

File: app/assets/stylesheets/pages/_home.scss (hunk 1 above)
Comment (Leprof20) [reply app/assets/stylesheets/pages/_home.scss:2]: Thanks, fixed!

File: app/assets/stylesheets/components/_footer.scss (hunk 2 above)
Comment (Leprof20) [reply app/assets/stylesheets/components/_footer.scss:38]: Thanks, fixed now

**Pull request merged**